schema/data/prod/energy_product.sql

```diff
@@ -0,0 +1,33 @@
+begin;
+
+alter table ggircs_portal.product disable trigger _protect_read_only_products;
+
+with rows as (
+insert into ggircs_portal.product(id, product_name, units, product_state, requires_emission_allocation, requires_product_amount, is_read_only)
+overriding system value
+values
+(35, 'Exported electricity', 'MWh', 'published' ,true, true, true),
```

Comment:
@matthieu-foucault should I have done this the other way around? With these energy products starting at id=1 and the test products being inserted after?

Reply:
@dleard yup :upside_down_face:

Reply:
ok yeah that makes more sense, I've swapped them.